### PR TITLE
Dilpreet | Test | Integrate mobile number library on voucher form 

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**
https://app.clickup.com/t/2p1fd62

### CODE REVIEW

**Comment by dilpreetsinghofficial** [comment]
Reviewed

File: apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html [hunk 1]
```diff
@@ -1012,17 +1012,40 @@
...
-                                                [placeholder]="localeData?.mobile_number_placeholder"
-                                                type="tel"
-                                                name="accountDetails.mobileNumber"
-                                                class="form-control attention-to"
-                                                [(ngModel)]="invFormData.accountDetails.mobileNumber"
-                                            />
+                                    <div class="col-sm-12 col-xl-9 col-lg-8 col-md-9 col-xs-12">
+                                        <div
+                                            class="col-sm-12 col-xl-6 col-lg-10 col-md-10 pl-0 ngx-wrapper"
+                                            *ngIf="isMobileNumber"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html]:
```suggestion
                                            *ngIf="showMobileNumberField"
```

File: angular.json [hunk 2]
```diff
@@ -114,8 +114,11 @@
                         "styles": [
                             "apps/web-giddh/src/assets/css/style-1.scss",
                             "apps/web-giddh/src/assets/css/style-2.scss",
-                            "node_modules/intl-tel-input/build/css/intlTelInput.css",
-                            "node_modules/ngx-lightbox/lightbox.css"
+                            "apps/web-giddh/src/assets/css/ladda-themeless.min.css",
```
Comment (ravinderwalkover) [inline angular.json]: check this

File: angular.json [hunk 3]
```diff
@@ -381,8 +384,11 @@
...
```
Comment (ravinderwalkover) [inline angular.json]: check this

File: apps/web-giddh/src/assets/locale/proforma-invoice/en.json [hunk 4]
```diff
@@ -107,5 +107,6 @@
     "exchange_label": "Exchange",
     "confirm_delete_file": "Are you sure you want to delete the attached file?",
     "enter_amount": "Please enter deposit amount",
-    "voucher_number_required": "Voucher number is mandatory."
+    "voucher_number_required": "Voucher number is mandatory.",
+    "invalid_contact_number": "Invalid Contact number"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/locale/proforma-invoice/en.json]:
```suggestion
    "invalid_mobile_number": "Invalid Mobile number"
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by ravinderwalkover — APPROVED** [review]

File: angular.json [hunk 5]
```diff
@@ -114,8 +114,8 @@
                         "styles": [
                             "apps/web-giddh/src/assets/css/style-1.scss",
                             "apps/web-giddh/src/assets/css/style-2.scss",
-                            "node_modules/intl-tel-input/build/css/intlTelInput.css",
-                            "node_modules/ngx-lightbox/lightbox.css"
```
Comment (ravinderwalkover) [inline angular.json:118]: why lightbox css is getting removed?

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by the-prateek-mishra — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**